mongomock/aggregate.py

```diff
@@ -238,7 +238,11 @@ def _handle_arithmetic_operator(self, operator, values):
             return math.sqrt(self.parse(values))
         if operator == '$subtract':
             assert len(values) == 2, 'subtract must have only 2 items'
-            res = self.parse(values[0]) - self.parse(values[1])
+            value_0 = self.parse(values[0])
+            value_1 = self.parse(values[1])
+            if isinstance(value_0, datetime.datetime) and isinstance(value_1, (int, float)):
```

Comment:
Let's not forget about `long` in Python 2. I suggest you use integer_types https://six.readthedocs.io/#six.integer_types

Reply:
Awesome. Done.